### PR TITLE
166816202 Signup email verification

### PR DESCRIPTION
**Pull request by vincentayorinde**
#### Description
This PR sends email verification message and token to newly registered users and allows each user to activate their account

#### Type of change
Please select the relevant option

- [ ] Bug fix (non-breaking changes which fixes an issue)
- [x] New feature (non-breaking changes which adds functionality)
- [ ] Breaking change (fix or feature that could cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Chore

#### Sample of Email
Screenshot 2019-06-29 at 5 45 30 PM

#### How Has This Been Tested?
please describe the test that you ran to verify your changes

- [ ] End to end 
- [ ] Integration
- [x] Unit Test
#### Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have linted my code prior to submission
- [ ] I have made corresponding changes to the documentation
- [x] My changes generates no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

#### Pivotal Tracker
[#166816202](https://www.pivotaltracker.com/story/show/166816202)

### CODE REVIEW

File: routes/v1/users.js [hunk 1]
```diff
@@ -6,5 +6,6 @@ const router = express.Router();
 
 router.post('/signup', Validation.signUp, Controller.signUp);
 router.post('/login', Validation.logIn, Controller.logIn);
+router.put(`/activate/:token`, Controller.activate);
```
[the remark this answers is not in the document]
Comment (andela-johia) [reply routes/v1/users.js]: Can you change the import from `Controller` to `User`? `Controller` does not sound right to me

[the remark this answers is not in the document]
Comment (andela-johia) [reply routes/v1/users.js]: Do it for the other routes as well

**Review by andela-johia — COMMENTED** [review]
It looks good just resolve the little feedback I gave

File: routes/v1/users.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (vincentayorinde) [reply routes/v1/users.js]: Alright. thank you for your feedback. I will work on this right away and update the PR

**Pull request merged**